inc/class-cachify.php

```diff
@@ -696,7 +701,7 @@ public static function process_flush_request( $data ) {
 		}
 
 		/* Check nonce */
-		if ( empty( $_GET['_wpnonce'] ) || ! wp_verify_nonce( $_GET['_wpnonce'], '_cachify__flush_nonce' ) ) {
+		if ( empty( $_GET['_wpnonce'] ) || ! wp_verify_nonce( sanitize_text_field( wp_unslash( $_GET['_wpnonce'] ) ), '_cachify__flush_nonce' ) ) {
```

Comment:
Nonce generation is luggable behavior, so it _might_ contain characters that do not pass `sanitize_text_field()`.

Reply:
I changed that to `sanitize_key`. There is a long discussion on https://github.com/WordPress/WordPress-Coding-Standards/issues/414 and https://github.com/WordPress/WordPress-Coding-Standards/issues/869 whether WPCS should throw an error at all and whether `sanitize_key` should be used or not. Alternative would be to add another ignore rule for that.

Reply:
I am aware of the discussions and totally fine with `sanitize_key`.